### PR TITLE
Nix flake meta

### PR DESCRIPTION
**Pull request by kloenk**
This adds a meta attribute to the nix package, so it can pass the search.nixos.org flake index tool

### CODE REVIEW

**Review by timoreo22 — APPROVED** [review]
Looks good !

**Comment by kloenk** [comment]
Also just verified it still builds under nixos x86_64-linux

**Review by d-513 — APPROVED** [review]

File: packages/nix/polymc/default.nix [hunk 1]
```diff
@@ -79,4 +79,17 @@ mkDerivation rec {
       --set GAME_LIBRARY_PATH ${gameLibraryPath} \
       --prefix PATH : ${lib.makeBinPath [ xorg.xrandr ]}
   '';
+
+  meta = with lib; {
+    homepage = "https://polymc.org/";
```
Comment (txtsd) [inline packages/nix/polymc/default.nix:84]: Is this the URL we want right now?

Comment (d-513) [reply packages/nix/polymc/default.nix:84]: 1.1.0 will be released after domain transfer anyways so let's keep that

**Comment by d-513** [comment]
@cidkidnix can you check this too

**Review by txtsd — APPROVED** [review]

**Review by Scrumplex — APPROVED** [review]

**Review by cidkidnix — APPROVED** [review]
:+1:

**Pull request merged**